cleanrl/dqn.py

```diff
@@ -49,17 +49,17 @@ def parse_args():
         help="the timesteps it takes to update the target network")
     parser.add_argument("--max-grad-norm", type=float, default=0.5,
         help="the maximum norm for the gradient clipping")
-    parser.add_argument("--batch-size", type=int, default=32,
+    parser.add_argument("--batch-size", type=int, default=128,
         help="the batch size of sample from the reply memory")
     parser.add_argument("--start-e", type=float, default=1,
         help="the starting epsilon for exploration")
     parser.add_argument("--end-e", type=float, default=0.05,
         help="the ending epsilon for exploration")
-    parser.add_argument("--exploration-fraction", type=float, default=0.8,
+    parser.add_argument("--exploration-fraction", type=float, default=0.5,
         help="the fraction of `total-timesteps` it takes from start-e to go end-e")
     parser.add_argument("--learning-starts", type=int, default=10000,
         help="timestep to start learning")
-    parser.add_argument("--train-frequency", type=int, default=1,
+    parser.add_argument("--train-frequency", type=int, default=10,
```

Comment:
Shouldn't DQN update after each environemnt step ?

Reply:
See the DQN paper

<img width="1451" alt="Screen Shot 2022-04-19 at 10 38 45 PM" src="https://user-images.githubusercontent.com/5555347/164135989-514a1d2c-36bb-4d15-9e4e-f95ef1ff0dd4.png">
 
And in C51 paper: 

"While our training regime closely follows that of DQN (Mnih et al., 2015),..."

Reply:
I see. Why 10 in this case ?

Reply:
Just a tuned one tbh

Reply:
Since it works never the less, looks good to me.

Reply:
Another motivation for this is to collect more data before learning, similar to how it works in ppo. But yeah it’s just a hyper parameter and seems to work well.